modules/proc_ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries.adoc

```diff
@@ -0,0 +1,23 @@
+:_content-type: PROCEDURE
+[id="ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries_{context}"]
```

Comment:
Suggest renaming the file to match the id, so `modules/ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries.adoc`.

Reply:
Updated as suggested.